### ISSUE TITLE
RFE: device path: bus/addr/port 

### ISSUE THREAD

**Issue by Andrei-Pozolotin** [nyholku/purejavahidapi#58]
please expose `libusb/usb4java` compatible device path: `bus/addr/port`

for example,  see: https://github.com/usb4java/usb4java-javax/issues/10

**Comment by Andrei-Pozolotin** [comment]
you expose `public String getDeviceId()`
https://github.com/nyholku/purejavahidapi/blob/master/src/purejavahidapi/HidDeviceInfo.java#L159

which looks like `/dev/bus/usb/${bus}/${addr}`

and is conceptually identical to `libusb` parameters `${bus}/${addr}`

so essentially I am asking for unified platform-agnostic `${bus}/${addr}` extract

**Comment by nyholku** [comment]
Yes, but the 'agnostic' is the problem, I do not see how to implement that.
